README.md

```diff
@@ -129,7 +135,19 @@ Let's say the Posts in the previous question are available at `http://localhost:
   3. If Step 1 is unsuccessful, print an error message to the console
 
 ```js
-// Your answer goes here...
+$(document).ready(()=>{
+  $(".get").on("click", () => {
+    $.ajax({
+      type: 'GET',
+      dataType: 'json',
+      url: "/posts"
```

Comment:
This line will work if this Javascript is part of your Rails app. If not, you'll need to specify `localhost:3000`